static/src/javascripts/components/bean/bean.js

```diff
@@ -106,7 +106,7 @@
               'fromElement offsetX offsetY pageX pageY screenX screenY toElement'))
           , mouseWheelProps = mouseProps.concat(str2arr('wheelDelta wheelDeltaX wheelDeltaY wheelDeltaZ ' +
               'axis')) // 'axis' is FF specific
-          , keyProps     = commonProps.concat(str2arr('char charCode key keyCode keyIdentifier '          +
+          , keyProps     = commonProps.concat(str2arr('char charCode key keyCode '          +
```

Comment:
We use this library in membership as well, it might be worth pushing this upstream or just plain forking it before we diverge further.